src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/Login.vue

```diff
@@ -6,11 +6,10 @@
     </PageContainer>
   </v-sheet>
   <PageContainer :margin-top="false">
-    <div class="d-flex flex-column ga-3 mb-10">
-      <Banner
-        type="info"
-        title="We're working to resolve technical issues affecting certificate information in the My ECE Registry platform. Recent users are asked to review their profiles for accuracy and contact the ECE Registry at 1-888-338-6622 if there are any issues."
-      />
+    <div v-for="(systemMessage, index) in configStore.systemMessages">
```

Comment:
could we get rid of the unused variable index since we're not using it? Happens in a few places.